constraints.txt

```diff
@@ -55,14 +56,15 @@ executing==1.2.0
 fastjsonschema==2.17.1
 fastobo==0.12.2
 filelock==3.12.2
-fonttools==4.40.0
+flake8==6.0.0
+fonttools==4.41.0
 fqdn==1.5.1
-funowl==0.1.13
+funowl==0.1.12
```

Comment:
victim 2

Reply:
caused by `oaklib` - I will look into this.